### PR TITLE
fix issue #13666

### PR DESCRIPTION
**Pull request by codehimanshu**
Bind off click event on change password button

Signed-off-by: Himanshu Agrawal <himanshuagrawal1998@gmail.com>

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by codehimanshu** [comment]
The -s option in commit command adds my username (not complete name) in Sign Off, so I manually enter the Signed Off By line in commit message. How should I automate it? Also the DCO test did not pass because it is expecting username instead of Full Name.

**Comment by mauriciofauth** [comment]
Hi, @codehimanshu.
You can change this using `git config` command:
```
git config --global user.name "Himanshu Agrawal"
git config --global user.email himanshuagrawal1998@gmail.com
```
If you want to change these settings only for phpMyAdmin, just remove the `--global` option.

File: js/functions.js [hunk 1]
```diff
@@ -3191,13 +3185,19 @@ AJAX.registerOnload('functions.js', function () {
...
+    $(document).off('click', '#change_password_anchor.ajax');
+
+    /**
      * Attach Ajax event handler on the change password anchor
      */
+
     $(document).on('click', '#change_password_anchor.ajax', function (event) {
         event.preventDefault();
-
+        console.log("#");
```
Comment (mauriciofauth) [inline js/functions.js]: This line seems to serve only for debugging. Did you intentionally leave it?

File: js/functions.js [hunk 2]
```diff
@@ -3191,13 +3185,19 @@ AJAX.registerOnload('functions.js', function () {
...
         var $msgbox = PMA_ajaxShowMessage();
-
+        // $('#change_password_anchor.ajax').bind("click");
```
Comment (mauriciofauth) [inline js/functions.js]: This line of code is commented out. If it's not useful, you can remove it.

File: js/functions.js (hunk 1 above)
Comment (codehimanshu) [reply js/functions.js]: Sorry, by mistake. Let me remove this.

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**